src/exercise/01.js

```diff
@@ -3,27 +3,25 @@
 
 import * as React from 'react'
 
-function Greeting() {
-  // 💣 delete this variable declaration and replace it with a React.useState call
-  const name = ''
+function Greeting({initialName = ''}) {
```

Comment:
I like how you navigated around the props... it's kindof clever... BUT... it's probably worth explicitly still using `props` here and below so you get the hang of callin' 'em props... so something like:

``
```suggestion
function Greeting(props) {
```

with a later....